src/gpt_review/_ask.py

```diff
@@ -20,9 +21,36 @@
 DEFAULT_KEY_VAULT = "https://dciborow-openai.vault.azure.net/"
 
 
-def _ask(question, max_tokens=100):
+def validate_parameter_range(namespace):
```

Comment:
```suggestion
def _validate_parameter_range(namespace):
```

Let's mark this as private to show we aren't intending for people to call this method in their own code.